bundles/org.openhab.binding.daikin/src/main/java/org/openhab/binding/daikin/internal/handler/DaikinAcUnitHandler.java

```diff
@@ -235,6 +250,16 @@ private void pollAirbaseStatus() throws IOException {
             updateTemperatureChannel(DaikinBindingConstants.CHANNEL_AC_TEMP, controlInfo.temp);
             updateState(DaikinBindingConstants.CHANNEL_AC_MODE, new StringType(controlInfo.mode.name()));
             updateState(DaikinBindingConstants.CHANNEL_AIRBASE_AC_FAN_SPEED, new StringType(controlInfo.fanSpeed.name()));
+
+            if (controlInfo.power == false) {
+                updateState(DaikinBindingConstants.CHANNEL_AC_HOMEKITMODE, new StringType("off"));
+            } else if (controlInfo.mode.name() == "COLD") {
```

Comment:
As also suggested:
```suggestion
            } else if (controlInfo.mode == Mode.COLD) {
```

In Java never compare strings with `==`. If you need to compare Strings use `equals`. But in this case it can be comparing enums. And enums can be compared with `==`.